### PR TITLE
add websockets

### PR DESCRIPTION
**Pull request by joel99**

### CODE REVIEW

File: server/typeDefs.ts [hunk 1]
```diff
@@ -1,12 +1,15 @@
 import * as fs from "fs";
 import * as path from "path";
 import { mediaAPI } from './plugins';
+import SocketPlugin from "./plugins/Socket";
 import { lowerSnake } from './common';
 
+export const SOCKETIO_KEY = "Socketio";
```
Comment (rahulrajus) [inline server/typeDefs.ts]: should be environment variable?

File: server/app.ts [hunk 2]
```diff
@@ -48,6 +68,9 @@ const resolvers = {
 	Query: {
 		get_messages: async (prev: any, args: any): Promise<IMessageReturn[]> => {
 			let plugin = args.plugin;
+			if (plugin === SOCKETIO_KEY) {
```
Comment (rahulrajus) [inline server/app.ts]: Also should be environment variable?

File: merged.graphql [hunk 3]
```diff
@@ -40,7 +40,8 @@ input PluginMaster {
 	slack: SlackConfig
 	twilio: TwilioConfig
 	twitter: TwitterConfig
-	appgt: AppGTConfig
+	f_c_m: FCMConfig
```
Comment (rahulrajus) [inline merged.graphql]: hm..I think the schema generator messed up the name for fcm. I can look into it.

File: server/app.ts (hunk 2 above)
Comment (joel99) [reply server/app.ts]: this one matters a little less

**Pull request merged**